ui/components/component-library/button-base/button-base.scss

```diff
@@ -1,7 +1,6 @@
 .mm-button-base {
   position: relative;
   height: 40px;
-  padding: 0; // TODO: remove once https://github.com/MetaMask/metamask-extension/pull/17006 is merged
```

Comment:
This was used because the Box didn't accept `0` values this has now been resolved in the TS version of the `Box` so it's no longer needed